### ISSUE TITLE
Limit protectionStones in nether

### ISSUE THREAD

**Issue by MatiRosen** [espidev/ProtectionStones#189]
Hi, i want to set the limit of protections in world:world to 4, and limit it in world_nether to 2. But i can't, if i put 4 protections in over, then i cant put them in nether.

**Comment by MatiRosen** [comment]
I need something like protectionstones.limit.world.[world].[amount]

**Comment by espidev** [comment]
You can try using per world permissions in a plugin like LuckPerms

**Comment by MatiRosen** [comment]
Yes. i tried with LuckPerms, i used: /lp group default permission set protectionstones.limit.4 true world=world and then  /lp group default permission set protectionstones.limit.2 true world=world_nether. But it doesn't work, it count the global protections so if you place 2 protections in the nether, then you can only place another two in overworld.
